### ISSUE TITLE
[SI-REVIEW-?]: 1.3.1 Info and Relationships - Appropriate landmark

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#945]
## Problem Description per Siteimprove

HTML5 and WAI-ARIA landmarks should be applied to the section of the page that corresponds to their type. This warning will always appear for your HTML5 and WAI-ARIA landmarks since a manual review is needed. 

## Page Description

- Top level page of a fresh Web Express install.
- A subpage with sidebar navigation

## What we're doing: 

Five landmark instances: 
1. Banner
2. Navigation (Desktop)
3. Navigation (Mobile)
4. Main
5. Aside (only on pages with a sidebar)
6. Footer

## Solution

Review the HTML5 or WAI-ARIA landmarks to make sure the landmark types matches the purpose of the sections they are applied to. Looks to me that they do.
